doc/classes/NavigationServer2D.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="NavigationServer2D" inherits="Object" is_experimental="true" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Server interface for low-level 2D navigation access.
+		A server interface for low-level 2D navigation access.
 	</brief_description>
 	<description>
-		NavigationServer2D is the server responsible for all 2D navigation. It handles several objects, namely maps, regions and agents.
-		Maps are made up of regions, which are made of navigation polygons. Together, they define the navigable areas in the 2D world.
-		[b]Note:[/b] Most NavigationServer changes take effect after the next physics frame and not immediately. This includes all changes made to maps, regions or agents by navigation related Nodes in the SceneTree or made through scripts.
+		NavigationServer2D is the server responsible for all 2D navigation. It handles navigation maps, regions and agents.
```

Comment:
NavigationServer is only responsible for navigation mesh based navigation and agent avoidance.
E.g. it does not handle point based navigation from the AStar2d/3D/Grid classes at all.